bowling/game.hpp

```diff
@@ -1,3 +1,6 @@
 #pragma once
 
-class Game {};
+class Game {
+    public:
+    static void printHelp();

```

Comment:
Could you explain me, why this function is static?

Reply:
@mateusz-talma Because in my opinion we don't need Game class object if user want's only read a help :)